packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -45,7 +45,7 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of items. If
    * the combo box is expanded, this will also scroll to the suggested item, and give it a selected style. Defaults to false.
    */
-  autoComplete?: boolean;
+  autoCompleteInput?: boolean;

```

Comment:
I don't think we can in this case. The term `autoComplete` IS actually defined on the HTMLAttribute interface, so technically it is still there, but its type has changed. `autoComplete` is normally a string `off` or `on` I think. We unfortunately hit this break in adding the `HTMLAttribute` inheritance, but we need that to inject html attributes into the element. The code was just added last week, I think we have to make this an exception.